lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -31,6 +32,14 @@ export class LhcPeriodsOverviewModel extends Observable {
         this._pagination.itemsPerPageSelector$.observe(() => this.notify());
 
         this._lhcPeriods = RemoteData.NotAsked();
+
+        this._namesFilterModel = new TextTokensFilterModel(
+            /^(([Ll][Hh][Cc]\d\d[A-Za-z]+)|([ ,;]+))*([Ll][Hh][Cc]\d\d[A-Za-z]+)(([ ,;]+)|([Ll][Hh][Cc]\d\d[A-Za-z]+))*$/,
```

Comment:
I'm pretty sure this can be simplified
Are you sure you want to be so strict? Why not simply allowing any string, it will simply match nothing

Reply:
after discussion, no constraints